privacyidea/lib/auditmodules/sqlaudit.py

```diff
@@ -516,8 +531,11 @@ def clear(self):
         self.session.commit()
     
     def audit_entry_to_dict(self, audit_entry):
-        sig = self.sign_object.verify(self._log_to_string(audit_entry),
-                                      audit_entry.signature)
+        if self.no_sign:
+            sig = None
```

Comment:
So setting ``PI_AUDIT_NO_SIGN = True`` would not only turn off signing, but also verification of signatures of already existing entries. I think this behavior makes sense, but we should add a line to the documentation.

Reply:
done